tsdb/points.go

```diff
@@ -52,6 +54,9 @@ type point struct {
 
 	// binary encoded field data
 	data []byte
+
+	// precision is set
+	precision string
```

Comment:
Do we really need to hold onto this?  It's only used during parsing and then thrown away.  Doesn't seem necessary to make it a field.